library/common/config/config.cc

```diff
@@ -228,83 +228,98 @@ const char* config_template = R"(
             budget_percent:
               value: 100
             min_retry_concurrency: 0xffffffff # uint32 max
+    outlier_detection: &base_outlier_detection
+      consecutive_5xx: 3
+      base_ejection_time: 0.001s
+      max_ejection_time: 0.001s
```

Comment:
At a high level this seems reasonable to me, however I'm not sure if there are any implications of such a short base/max ejection time. At minimum I would add more comments here as it's unclear to the reader why it's configured this way. Also, please do a bunch of manual testing, especially in totally offline modes.

Reply:
Yep, makes sense. I did some very rudimentary validation going into the weekend, but I'll do some more thorough testing today/tomorrow.